package.json

```diff
@@ -3,13 +3,20 @@
   "module": "index.ts",
   "type": "module",
   "scripts": {
-    "start": "bun run src/index.ts"
+    "start": "bun run src/index.ts",
+    "format:check": "prettier -c \"src/**/*.ts\"",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint:check": "eslint \"src/**/*.ts\"",
+    "lint": "yarn lint:check --fix"
   },
   "devDependencies": {
     "@types/node-telegram-bot-api": "^0.64.3",
-    "bun-types": "latest"
-  },
-  "peerDependencies": {
```

Comment:
Why not in the dev dependencies?

Reply:
Sorry, which one? If typescript, it was in peerDependencies, since this is not a library, thought it should be moved to devDependencies

Reply:
My bad I thought you moved  the DevDependencies to PeerDependencies...